PhysicsTools/PatAlgos/python/selectionLayer1/selectedPatCandidates_cff.py

```diff
@@ -36,5 +36,8 @@
 selectedPatCandidates = cms.Sequence(selectedPatCandidateSummary, selectedPatCandidatesTask)
 
 from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+#from Configuration.Eras.Modifier_pp_on_PbPb_run3_cff import pp_on_PbPb_run3
```

Comment:
Shouldn't the commented out part replace the uncommented one below, now that you are considering separately `pp_on_AA_2018` and `pp_on_PbPb_run3`?